pkg/controllers/machine_controller.go

```diff
@@ -173,6 +185,33 @@ func (r *MachineReconciler) Start(ctx context.Context) error {
 	return nil
 }
 
+func (r *MachineReconciler) startCheckAndEnqueueMachineStatus(ctx context.Context, log logr.Logger) {
+	wait.UntilWithContext(ctx, func(ctx context.Context) {
+		machines, err := r.machines.List(ctx)
+		if err != nil {
+			log.Error(err, "failed to list machines")
+			return
+		}
+
+		for _, machine := range machines {
+			if machine.DeletedAt != nil || !slices.Contains(machine.Finalizers, MachineFinalizer) {
+				continue
+			}
+			id := machine.ID
+
+			state, err := r.getMachineState(id)
+			if err != nil {
+				log.Error(err, "failed to get machine state", "machine:", id)
```

Comment:
key is without : and pls unified name of key to machineId

Reply:
Could you be please more clear on this?

Reply:
```suggestion
				log.Error(err, "failed to get machine state", "machineId", id)
```

Reply:
i don't see more log message now. so i don't see any benefits in create new logger.